lib/completion-templates.ts

```diff
@@ -5,45 +5,47 @@ export const completionShTemplate = `###-begin-{{app_name}}-completions-###
 # Installation: {{app_path}} {{completion_command}} >> ~/.bashrc
 #    or {{app_path}} {{completion_command}} >> ~/.bash_profile on OSX.
 #
-_{{app_name}}_yargs_completions()
+_{{safe_app_name}}_yargs_completions()
```

Comment:
`safe_app_name` is limited to characters that are safe for a bash function name.